src/frontends/tensorflow/src/op/concat.cpp

```diff
@@ -34,17 +34,47 @@ OutputVector translate_concat_op(const NodeContext& node) {
     get_const_input(node, axis_idx, &tf_concat_axis_vec);
     int64_t concat_axis = tf_concat_axis_vec[0];
```

Comment:
please add one more check that `tf_concat_axis_vec.size() > 0`

Reply:
Done